server/src/main/java/org/eclipse/openvsx/UpstreamRegistryService.java

```diff
@@ -252,33 +237,57 @@ public QueryResultJson query(QueryParamJson param) {
                 "targetPlatform", param.targetPlatform
         );
 
+        var queryString = queryParams.entrySet().stream()
+            .filter(entry -> !Strings.isNullOrEmpty(entry.getValue()))
+            .map(entry -> entry.getKey())
+            .map(key -> key + "={" + key + "}")
```

Comment:
Shouldn't this be `.map(key -> key + "={" + value + "}")`

Reply:
No, the key is used so that the template is uniform. This is to group all `/api/-/query` requests together. RestTemplate uses `queryParams` to fill in the template before making the request.